components/components-definition.json

```diff
@@ -72,6 +72,8 @@
             "icon": "icons/components/slideshow.svg",
             "label": "{{ 'DESIGN_COMPONENT_SLIDESHOW_LABEL' | translate }}",
             "properties": [
+                { "name": "slides", "directiveKey": "slideshow" },
+                { "control": { "type": "header" }, "label": "Slideshow properties" },
```

Comment:
Shouldn't this be a string key instead?

Reply:
String key?

Reply:
Like `"label": "{{ 'DESIGN_COMPOMNENT_SLIDESHOW_PROPERTIES' | translate }}"`

Reply:
Added as bullet to CSH-3722. There is no TMS string yet.